packages/celotool/src/e2e-tests/validator_order_tests.ts

```diff
@@ -0,0 +1,109 @@
+import { ContractKit, newKit } from '@celo/contractkit'
+import { assert } from 'chai'
+import _ from 'lodash'
+import { getContext, GethTestConfig, sleep } from './utils'
+
+const validatorAddress = '0x47e172f6cfb6c7d01c1574fa3e2be7cc73269d95'
+// @ts-ignore
+const validatorAddresses = [
+  '0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95',
+  '0xa0Af2E71cECc248f4a7fD606F203467B500Dd53B',
+  '0xA42c9b0d1A30722AEa8b81E72957134897E7A11a',
+  '0x848920B14154b6508B8d98e7eE8159AA84b579A4',
+  '0x2ffe970257D93eae9d6B134f528b93b262C31030',
+]
+
+const VALIDATORS = 5
+const EPOCH = 10
+const EPOCHS_TO_WAIT = 5
+const BLOCK_COUNT = EPOCH * EPOCHS_TO_WAIT
+const VALIDATOR_BLOCKS = BLOCK_COUNT / VALIDATORS
+const BLOCK_COUNT_ARRAY = Array.from(Array(BLOCK_COUNT).keys())
```

Comment:
Nit: This took me a momment to understand. `_.range(BLOCK_COUNT)` constructed in-line where you need it would be a bit more readable.